conf/test.config

```diff
@@ -25,5 +25,5 @@ params {
   // Genome references
   fasta = 'https://github.com/nf-core/test-datasets/raw/rnaseq/reference/genome.fa'
   gtf = 'https://github.com/nf-core/test-datasets/raw/rnaseq/reference/genes.gtf'
-  transcriptome = 'https://github.com/nf-core/test-datasets/raw/rnaseq/reference/transcriptome.fasta'
+  transcript_fasta = 'https://github.com/nf-core/test-datasets/raw/rnaseq/reference/transcriptome.fasta'
```

Comment:
I dont think we need this anymore because we should be able to build it from `--fasta` and `--gtf`

Reply:
Can remove `transcript_fasta` from `test.config` because it will be built be the pipeline.

Reply:
We do have sometimes multiple test profiles, e.g. I tend to have one for each aligner or method in the pipeline and then setting everything in that test profile there. Makes it easier than having multiple very long configuration lines in the `.travis.yml` so I think it would be great to have that here too 👍